Resources/translations/messages.en.xliff

```diff
@@ -10,6 +10,18 @@
         <source>dummy</source>
         <target>dummy</target>
       </trans-unit>
+      <trans-unit id="0b5e2fc2fbb589463a61d536b0d97fa25028bbfd" resname="notification.spam_complaint_received">
+        <source>notification.spam_complaint_received</source>
```

Comment:
please add the `state="new"` for all new translations

Reply:
I've just updated my (currently Symfony 3.3)  project, and now every page is throwing the error:

An exception has been thrown during the rendering of a template 
("Invalid resource provided: "1.2"; Errors: [ERROR 1867] 
Element '{urn:oasis:names:tc:xliff:document:1.2}
source', attribute 'state': The attribute 'state' is not allowed. (in /home/.../web/ - line 14, column 0)

The [Xliff schema](http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#state) does  mention 'state', but it's not on `source`, its on the next element - `target`.

Reply:
ups sorry. I'll fix that later today.